api/src/main/java/edu/cornell/mannlib/vitro/webapp/controller/json/JsonServlet.java

```diff
@@ -26,131 +26,139 @@
 import edu.cornell.mannlib.vitro.webapp.utils.log.LogUtils;
 
 /**
- * This servlet is for servicing requests for JSON objects/data.
- * It could be generalized to get other types of data ex. XML, HTML etc
+ * This servlet is for servicing requests for JSON objects/data. It could be
+ * generalized to get other types of data ex. XML, HTML etc
+ * 
  * @author bdc34
  *
- * Moved most of the logic into a group of JsonProducer classes. jeb228
+ *         Moved most of the logic into a group of JsonProducer classes. jeb228
  */
-@WebServlet(name = "JSONService", urlPatterns = {"/dataservice"} )
+@WebServlet(name = "JSONService", urlPatterns = { "/dataservice" })
 public class JsonServlet extends VitroHttpServlet {
     private static final long serialVersionUID = 1L;
     private static final Log log = LogFactory.getLog(JsonServlet.class);
-    private static final int INDIVIDUALS_PER_PAGE = 30;
+
+    /*
+     * Approximately 50 SPARQL queries are required to process each individual. 
+     * The value of this variable has a major impact on the refresh 
+     * performance of the front pages of each tab, especially on person and organization
+     *
+     * I suggest to put the assignment of this variable in runtime.properties
+     * 
+     */
+    private static final int INDIVIDUALS_PER_PAGE = 15;
```

Comment:
Not sure it should be in runtime.properties or it there could be some other place to configure that, but changing default value for all Vitro and VIVO users doesn't look good to me.